src/languages/en-US/system.json

```diff
@@ -7,7 +7,6 @@
 	"exceededLengthOutputTime": "{{time}}",
 	"exceededLengthOutputFile": "Sent the result as a file.",
 	"exceededLengthOutputHastebin": "Sent the result to hastebin: {{url}}",
-	"externalServerError": "The external service we use could not process our message. Please try again later.",
```

Comment:
?

Reply:
I also have no idea when or where did this line come from, but as of this PR, Skyra's reliance to external services/APIs are at a minimum. Either way many commands that did implemented its own lines.